cob_moveit_config/raw3-1/config/kinematics.yaml

```diff
@@ -1,16 +1,11 @@
 arm:
   kinematics_solver: kdl_kinematics_plugin/KDLKinematicsPlugin
```

Comment:
Nor working for now:

Kinematics solver of type 'ur_kinematics/UR10KinematicsPlugin' could not be initialized for group 'arm'

We keep the previous solver.